src/components/CubeOverviewModal.js

```diff
@@ -368,7 +368,7 @@ class CubeOverviewModal extends Component {
                 <h6>Description</h6>
                 <TextEntry
                   name="blog"
-                  value={this.state.cube.descriptionhtml ? this.state.cube.descriptionhtml : ''}
+                  value={cube.descriptionhtml && cube.descriptionhtml !== 'undefined' ? cube.descriptionhtml :cube.description}
```

Comment:
```suggestion
                  value={typeof cube.descriptionhtml !== 'undefined' ? cube.descriptionhtml :cube.description}
```

Reply:
What is this hack

Reply:
This is actually how you are supposed to check if something is undefined

Reply:
The issue is descriptionhtml can be set to the string value of 'undefined' if any other field was updated with the new modal

Reply:
So this checks if it's actually undefined, or if it is equal to the string value of undefined